cookbook/core/type_system/structured_dataset.py

```diff
@@ -87,18 +87,80 @@ def get_subset_df(
 ) -> Annotated[StructuredDataset, subset_cols]:
     df = df.open(pd.DataFrame).all()
     df = pd.concat([df, pd.DataFrame([[30]], columns=["Age"])])
-    # On specifying BigQuery uri for StructuredDataset, Flytekit will write pd.dataframe to a BigQuery table
+    # On specifying BigQuery uri for StructuredDataset, flytekit writes pd.dataframe to a BigQuery table
     return StructuredDataset(dataframe=df)
 
 
 # %%
+# StructuredDataset with ``uri`` Argument
+# ========================================
+#
+# BigQuery ``uri`` allows you to load and retrieve data from cloud using the ``uri``. The ``uri`` comprises of the bucket name and the filename prefixed with ``gs://``.
```

Comment:
``uri`` doesn't mean you only give BigQuery uri. It could be an s3 or a gcs uri, too.